pkg/resources/management.cattle.io/v3/globalrole/GlobalRole.md

```diff
@@ -0,0 +1,8 @@
+## Validation Checks
+
+Note: all checks are bypassed if the GlobalRole is being deleted
+
+### Escalation Prevention
+
+Users can only change GlobalRoles with rights less than or equal to those they currently possess. This is to prevents privilege escalation. 
```

Comment:
```suggestion
Users can only change GlobalRoles with rights less than or equal to those they currently possess. This is to prevent privilege escalation. 
```

Reply:
Change Made.